Alignment/TrackerAlignment/plugins/AlignmentPrescaler.cc

```diff
@@ -169,15 +209,15 @@ void AlignmentPrescaler::produce(edm::Event& iEvent, const edm::EventSetup& iSet
       //      tmpflag.SetDetId(hit->geographicalId());
 
       if (isOverlapHit) {
-        //cout<<"  DetId="<<tmpdetid<<" is Overlap! "<<flush;
+        LogDebug("AlignmentPrescaler") << "  DetId=" << tmpdetid << " is Overlap! " << flush;
         takeit = (float(myrand_->Rndm()) <= overlapPrescFactor_);
       }
       if (!takeit) {
         float rr = float(myrand_->Rndm());
         takeit = (rr <= hitPrescFactor_);
       }
       if (takeit) {  //HIT TAKEN !
-        //cout<<"  DetId="<<tmpdetid<<" taken!"<<flush;
+        LogDebug("AlignmentPrescaler") << "  DetId=" << tmpdetid << " taken!" << flush;
```

Comment:
I think the `flush` is not needed, the edm::LogPrint will take care of it, no?

Reply:
Or add the `std::`, btw this means that there is an `using namespace std;` in the code, right? I thought that's discouraged